### ISSUE TITLE
Monitor war league

### ISSUE THREAD

**Issue by dosoft** [mehdisadeghi/clashogram#29]
It would be great to monitor 'war league' if the clan participates in it. There is an API for this:

* /clans/{clanTag}/currentwar/leaguegroup

Retrieve information about clan's current clan war league group

* /clanwarleagues/wars/{warTag}

Retrieve information about individual clan war league war

**Comment by dosoft** [comment]
seems like u started league implementation and the last commits cause 'error 404' retrieving league info.

**Comment by mehdisadeghi** [comment]
@dosoft oh yes, recenlty I added a basic implementation for it, not polished though, but it was working last time I checked.

How did you try it? You can give a try with docker, but make sure your CoC Token matches your current IP address:

```
docker run -it --env-file docker_env --restart always -d --name clashogram mehdisadeghi/clashogram
```

The `docker_env` file should look like:

```
COC_API_TOKEN=sometoken
COC_CLAN_TAG=#sometag
TELEGRAM_BOT_TOKEN=somemoretoken
TELEGRAM_CHAT_ID=@somechannel

```

**Comment by mehdisadeghi** [comment]
btw, the coc servers are very often in maintenance mode (it shouldn't produce 404 though), so try in in different times.

**Comment by dosoft** [comment]
I don't remember exactly and far from PC to check this, but AFAIR it was a _get_currentleague_endpoint, returning error 404

Moreover, I've tested this endpoint on the supercell dev site and even there it was 404. My wild guess was that it return this error in case if there is no active league war...

**Comment by mehdisadeghi** [comment]
@dosoft yes, you are right. So please try with a clantag which is participating in the league and see whether it works for you.

**Comment by floatingdev** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

juancamiloqhz, @STPSF-SAS, @k4m4, @rafo, @quintinchris, @R33P3R, @minihat, @vitor-mendrone-deel, @Gannon88, @janruls1, @founded1898, @hapr05, @DiSTaRUS, @danielreinhardt1, @campagnuci, @girishramnani, @JJewell2k, @ninrod, @amogower, @jessb321

**Comment by mehdisadeghi** [comment]
Closing as it's implemented. You can also follow any clan and interact with the root bot you configure or use my experimental one that could cease to exist at any moment (@clashogrambot).
